### ISSUE TITLE
why loss_bbox=0 and loss_dfl=0?

### ISSUE THREAD

**Issue by cena001plus** [PaddlePaddle/PaddleDetection#5137]
`[01/19 18:01:47] ppdet.engine INFO: Epoch: [0] [  60/605] learning_rate: 0.028000 loss_vfl: 0.187070 loss_bbox: 0.000000 loss_dfl: 0.000000 loss: 0.187070 eta: 3 days, 2:44:43 batch_cost: 3.6914 data_cost: 0.0013 ips: 6.5015 images/s

[01/19 18:02:58] ppdet.engine INFO: Epoch: [0] [  80/605] learning_rate: 0.034000 loss_vfl: 0.120860 loss_bbox: 0.000000 loss_dfl: 0.000000 loss: 0.120860 eta: 3 days, 14:17:00 batch_cost: 3.2840 data_cost: 0.9271 ips: 7.3082 images/s
`

**Comment by cena001plus** [comment]
solved!    

Because I modified the code, the code to read bbox from xml has error

ppdet/data/source/voc.py
